docs/public-networks/reference/api/index.md

```diff
@@ -2775,6 +2775,194 @@ curl -X POST -H "Content-Type: application/json" --data '{ "query": "{block (num
 
 <!--/tabs-->
 
+### `eth_getBlockReceipts`
+
+This JSON-RPC method returns all transaction receipts for a given block and takes the block identifier as a parameter. Transaction receipts are useful because they provide a way to track the success or failure of a transaction (`1` if successful and `0` if failed), as well as the amount of gas used and any event logs that might have been produced by a smart contract during the transaction.
+
+Transaction receipts also contain the transaction hash, the block number and hash in which the transaction was included, and the contract address of any contract created as a result of the transaction.
+
+#### Parameters
+
+- `blockNumber`: _string_ - hexadecimal or decimal integer representing a block number or one of the string tags `latest`, `earliest`, `pending`, `finalized`, or `safe`, as described in [block parameter](../../how-to/use-besu-api/json-rpc.md#block-parameter)
+
+- `verbose`: _boolean_ - if `true`, returns the full [transaction objects](objects.md#transaction-object); if `false`, returns only the hashes of the transactions.
+
+#### Returns
+
+`result`: _object_ - [block object](objects.md#block-object), or `null` when there is no block.
+
+<!--tabs-->
+
+# curl HTTP
+
+```bash
+curl -X POST "CHAINSTACK_ARCHIVE_NODE_URL" \
+  -H 'Content-Type: application/json' \
+  --data '{"method":"eth_getBlockReceipts","params":["latest"], "jsonrpc":"2.0","id":1}'
```

Comment:
```suggestion
curl -X POST --data '{"jsonrpc":"2.0","method":"eth_getBlockReceipts","params":["latest"],"id":1}' http://127.0.0.1:8545
```

Update to match the curl request examples for the other API methods on this reference page. Currently, they're all on one line and use the local RPC URL (`http://127.0.0.1:8545`).

Reply:
@mjsmike62 Looks like this change has reverted back to before the suggestion.

Reply:
Edit has been incorporated.